package.json

```diff
@@ -76,6 +76,7 @@
     "esbuild": "^0.18.16",
     "escape-string-regexp": "^5.0.0",
     "etag": "^1.8.1",
+    "firebase-functions": "^4.4.1",
```

Comment:
Should this be moved to devDependencies to have the correct types but still let the user install their own version of firebase-functions?

Reply:
Yes that seems good. (later with dedicated package, we can try to auto install however explicit dep is always better)

Reply:
@posva @pi0 I did try adding this as a dev dep, but came up against some build errors that were way above my head. See this comment in the original PR:
https://github.com/unjs/nitro/pull/1142#discussion_r1249622185

Reply:
Strange. I think it can be a regular dependency for users too. We really need to try merging one pr and try end-to-end via edge channel to resolve such possible issues.